spec.bs

```diff
@@ -1045,7 +1141,66 @@ Note: Previously, the `payment` extension allowed for the creation of
               enrollment.https.html
             </wpt>
 
-:  <dfn export>Client extension processing ([=authentication extension|authentication=])</dfn>
+    1. Before step 13 (creating a {{CollectedClientData}}).
+
+        1. Let |bbk_allowed_algorithms| be {{AuthenticationExtensionsPaymentInputs/browserBoundPubKeyCredParams}}.
+
+        1. Let (|bbk| - the platform specific keypair handle, |bbk_id| - an
+            identifier of the |bbk|) be the result of
+            {{BrowserBoundKeyStore/CreateKeyPair}}(|bbk_allowed_algorithms|).
+
+            <dl class="switch">
+
+                : BBK keypair was not created
+                :: Skip steps below relating to |bbk| and |bbk_public_key|.
+
+            </dl>
+
+        1. Let |bbk_public_key| be the result of
+            {{BrowserBoundKeyStore/GetBrowserBoundPublicKey}}(|bbk|).
+
+    1. In step 13, instead of creating the {{CollectedClientData}} create a {{CollectedClientPaymentData}} whose fields are:
+
+        1. {{CollectedClientPaymentData/payment}} set to a {{CollectedClientAdditionalPaymentData}} with
+
+            : {{CollectedClientAdditionalPaymentData/browserBoundPublicKey}}
+            :: |bbk_public_key| - The COSE_Key encoded public key.
+
+    1. In step 22 (webnauthn-3#CreateCred-async-loop), when "If any
```

Comment:
Could maybe just say "Just before creating a [PublicKeyCredential](https://w3c.github.io/webauthn/#publickeycredential), do [...]"

See also how WebAuthn has done this for https://w3c.github.io/webauthn/#sctn-appid-exclude-extension - albeit they added bookmarks into their spec to make their life easier, I think? (need to check out their bikeshed source)